pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -6,14 +6,50 @@
    "source": [
     "# Reactor Kinetics Example \n",
     "\n",
-    "Jialu Wang (jwang44@nd.edu) and Alex Dowling (adowling@nd.edu)\n",
+    "Jialu Wang (jwang44@nd.edu), Alex Dowling (adowling@nd.edu), and Hailey Lynch (hlynch@nd.edu)\n",
     "\n",
     "University of Notre Dame\n",
     "\n",
-    "This notebook conducts design of experiments for a reactor kinetics experiment with the Pyomo.DoE.\n",
+    "This notebook conducts design of experiments for a reactor kinetics experiment with Pyomo.DoE. \n",
```

Comment:
Change to "This notebook demonstrates the main features of Pyomo.DoE (model-based design of experiments) using a reactions kinetics example. See Wang and Dowling (2022), AIChE J., for more information". Make the references a hyperlink to here: https://aiche.onlinelibrary.wiley.com/doi/full/10.1002/aic.17813